javascript/283-Move-Zeroes.js

```diff
@@ -1,3 +1,23 @@
+/**
+ * Linear Time
+ * Time Complexity  O(N) | Space Complexity O(N);
+ * https://leetcode.com/problems/move-zeroes/
+ * @param {number[]} nums
+ * @return {void} Do not return anything, modify nums in-place instead.
+ */
+var moveZeroes = function(nums) {
+
+    const zeroAtTheEnd = Array(nums.length).fill(0);
+    let left = 0;
+    for (let i = 0; i < nums.length; i++) {
+        if (nums[i]) {
```

Comment:
You should define your if conditions

Reply:
@aakhtar3, What do you mean by defining if conditions?

Reply:
Look at the Space O(1) if condition